lib/finch/http1/conn.ex

```diff
@@ -50,29 +50,31 @@ defmodule Finch.Conn do
   end
 
   def transfer(conn, pid) do
-    case HTTP.controlling_process(conn.mint, pid) do
-      # HTTP.controlling_process causes a side-effect, but it doesn't actually
+    case HTTP1.controlling_process(conn.mint, pid) do
+      # HTTP1.controlling_process causes a side-effect, but it doesn't actually
       # change the conn, so we can ignore the value returned above.
       {:ok, _} -> {:ok, conn}
       {:error, error} -> {:error, conn, error}
     end
   end
 
   def open?(%{mint: nil}), do: false
-  def open?(%{mint: mint}), do: HTTP.open?(mint)
+  def open?(%{mint: mint}), do: HTTP1.open?(mint)
 
   def set_mode(conn, mode) when mode in [:active, :passive] do
-    case HTTP.set_mode(conn.mint, mode) do
+    case HTTP1.set_mode(conn.mint, mode) do
       {:ok, mint} -> {:ok, %{conn | mint: mint}}
       _ -> {:error, "Connection is dead"}
     end
   end
 
-  def stream(%{mint: nil}, _), do: {:error, "Connection is dead"}
+  def discard(%{mint: nil}, _), do: :unknown
```

Comment:
If there is no connection, then the message is certainly `:unknown` for this particular worker.